LICENSE

```diff
@@ -2,6 +2,8 @@ MIT License
 
 Copyright (c) 2018, MIT Data To AI Lab
 
+Copyright © 2022 DataCebo, Inc.
```

Comment:
Will the © character properly show up in the license? This is is exactly what the doc has specified, but I guess we should keep the character consistent -- use (c) in both places or © in both places

Reply:
Not sure, but I've seen `(c)` in other libraries so I'll do that one